src/usr/lib/rsetup/cli/wi-fi.sh

```diff
@@ -3,14 +3,32 @@
 ALLOWED_RCONFIG_FUNC+=("connect_wi-fi")
 
 connect_wi-fi() {
+    local i ssid="$1" password="$2" total_retry=10
+    local command=("nmcli device wifi connect $ssid")
     case $# in
-        1)
+        1|2)
             nmcli radio wifi on
-            nmcli dev wifi connect "$1"
-            ;;
-        2)
-            nmcli radio wifi on
-            nmcli dev wifi connect "$1" password "$2"
+
+            if (( $# == 2 ))
+            then
+                command+=(password "$password")
+            fi
+
+            for ((i = 0; i < total_retry; i++))
+            do
+                # shellcheck disable=SC2068
+                if ${command[@]}
```

Comment:
```suggestion
                if “${command[@]}”
```
凡是涉及到[@]引用数组的一概需要用引号